### PR TITLE
Release Flink 1.14.5

### PR DESCRIPTION
**Pull request by HuangXingBo**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -287,8 +287,8 @@ release_archive:
         release_date: 2022-05-05
       -
         version_short: "1.14"
-        version_long: 1.14.4
-        release_date: 2022-03-02
+        version_long: 1.14.5
```
Comment (knaufk) [inline _config.yml:290]: Here we should list all releases instead of replacing the last one.

File: _posts/2022-06-10-release-1.14.5.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+title:  "Apache Flink 1.14.5 Release Announcement"
+date: 2022-06-10T08:00:00.000Z
+categories: news
+authors:
+- xingbo:
+  name: "Xingbo Huang"
+
+excerpt: The Apache Flink Community is please to announce another bug fix release for Flink 1.14.
```
Comment (knaufk) [inline _posts/2022-06-10-release-1.14.5.md:10]:
```suggestion
excerpt: The Apache Flink Community is pleased to announce another bug fix release for Flink 1.14.
```

**Review by knaufk — CHANGES_REQUESTED** [review]
Generally, LGTM. Just two comments.

**Comment by HuangXingBo** [comment]
Thanks @knaufk for the review. I have resolved the comments at the latest commit.

File: _posts/2022-06-10-release-1.14.5.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+---
+
+The Apache Flink Community is please to announce another bug fix release for Flink 1.14.
```
Comment (dianfu) [inline _posts/2022-06-10-release-1.14.5.md:14]:
```suggestion
The Apache Flink Community is pleased to announce another bug fix release for Flink 1.14.
```

**Review by dianfu — APPROVED** [review]
LGTM with just one minor comment.

**Comment by shuiqiangchen** [comment]
LGTM overall

File: _posts/2022-06-10-release-1.14.5.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+This release includes 67 bug and vulnerability fixes and minor improvements for Flink 1.14.
```
Comment (gaoyunhaii) [inline _posts/2022-06-10-release-1.14.5.md:16]:
```suggestion
This release includes 67 bugs, vulnerability fixes and minor improvements for Flink 1.14.
```

**Pull request closed without merging**